includes/to-post-type.php

```diff
@@ -446,6 +446,7 @@ function cf_custom_fields_capture_entry($config, $form){
 		 * @param array $fie;d Field config @since 2.2.0
 		 * @param array $form Form config @since 2.2.0
 		 */
+		$value = Caldera_Forms::get_field_data( $value, $form ); 
```

Comment:
@New0 I would have used Caldera_Forms::do_magic_tags( $value, $form ); here, because then any  magic tag should work. 

Did you try that?

Reply:
I tested it and it works with do_magic_tags()